misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -87,123 +85,101 @@ public static void mergeApiDocs(String apiDocsRoot)  {
-                                jsonPackageLib.packages.get(0).name + "/latest\">link</a> </h1>\n" +
-                                "</body>\n" +
-                                "</html>";
-                        FileUtils.write(newIndex, htmlData, StandardCharsets.UTF_8, false);
-                    } catch (IOException e) {
-                        String errorMsg = String.format("API documentation generation failed. Cause: %s",
-                                                        e.getMessage());
-                        out.println(errorMsg);
-                        log.error(errorMsg, e);
-                        return;
                     }
                 }
+                }
             }
         }
-        writeAPIDocs(packageLib, apiDocsRoot, true);
+        packageLib.packages.sort((o1, o2) -> o1.name.compareToIgnoreCase(o2.name));
+        writeAPIDocs(packageLib, apiDocsRoot, true, false);
 
-        // Create the central json
-        String json = gson.toJson(centralLib);
-        File jsonFile = new File(apiDocsRoot + File.separator + DATA_DIR + File.separator + CENTRAL_DOC_DATA_JSON);
-        try (java.io.Writer writer = new OutputStreamWriter(new FileOutputStream(jsonFile),
+        // Create the central standard library index json
+        String stdIndexJson = gson.toJson(centralLib);
+        File stdIndexJsonFile = new File(apiDocsRoot + File.separator + CENTRAL_STDLIB_INDEX_JSON);
+        try (java.io.Writer writer = new OutputStreamWriter(new FileOutputStream(stdIndexJsonFile),
                 StandardCharsets.UTF_8)) {
-            writer.write(new String(json.getBytes(StandardCharsets.UTF_8), StandardCharsets.UTF_8));
+            writer.write(new String(stdIndexJson.getBytes(StandardCharsets.UTF_8), StandardCharsets.UTF_8));
+        } catch (IOException e) {
+            out.printf("docerina: failed to create the %s. Cause: %s%n", CENTRAL_STDLIB_INDEX_JSON, e.getMessage());
+            log.error("Failed to create {} file.", CENTRAL_STDLIB_INDEX_JSON, e);
+        }
+
+        // Create the central standard library search json
+        String stdSearchJson = gson.toJson(genSearchJson(packageLib));
+        File stdSearchJsonFile = new File(apiDocsRoot + File.separator + CENTRAL_STDLIB_SEARCH_JSON);
+        try (java.io.Writer writer = new OutputStreamWriter(new FileOutputStream(stdSearchJsonFile),
+                StandardCharsets.UTF_8)) {
+            writer.write(new String(stdSearchJson.getBytes(StandardCharsets.UTF_8), StandardCharsets.UTF_8));
         } catch (IOException e) {
-            out.printf("docerina: failed to create the %s. Cause: %s%n", CENTRAL_DOC_DATA_JSON, e.getMessage());
-            log.error("Failed to create {} file.", CENTRAL_DOC_DATA_JSON, e);
+            out.printf("docerina: failed to create the %s. Cause: %s%n", CENTRAL_STDLIB_SEARCH_JSON, e.getMessage());
+            log.error("Failed to create {} file.", CENTRAL_STDLIB_SEARCH_JSON, e);
```

Comment:
Why use 2 types of logging ?

Reply:
Removed the out.println logging in all error occurrences.